### PR TITLE
Add main menu to Sprity Bird

### PR DESCRIPTION
**Pull request by heyx3**
@JamesMcMahon

### CODE REVIEW

File: spritybird/Classes/AppDelegate.m [hunk 1]
```diff
@@ -15,7 +15,8 @@ @implementation AppDelegate
 #pragma mark - Skillz SDK Delegate implementation
 - (void)tournamentWillBegin:(NSDictionary *)gameParameters
 {
-    ViewController* viewController = (ViewController*) self.window.rootViewController;
+    ViewController* viewController = (ViewController*) self.window.rootViewController.modalViewController;
+    //self.window.
```
Comment (JamesMcMahon) [inline spritybird/Classes/AppDelegate.m]: Can we remove commented out code?

**Comment by JamesMcMahon** [comment]
Just 1 nitpick above.

Can we fix that and collapse this branch into a single commit (in order to keep the commit history clean)?

File: spritybird/Classes/AppDelegate.m (hunk 1 above)
Comment (heyx3) [inline spritybird/Classes/AppDelegate.m]: Sorry, I don't know how that got in there.

**Comment by heyx3** [comment]
Replaced with https://github.com/skillz/SprityBird/pull/3

**Pull request closed without merging**